test/utils_test.js

```diff
@@ -241,6 +241,26 @@ describe("utils", () => {
           foo: 42,
         });
       });
+
+      it("should fill array with additional items schema when items is empty", () => {
```

Comment:
This is not a great test description, but I'm not sure what a better one would be.

Reply:
I think it is quite ok, it describes what it is testing.